package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Command line interface for JavaAutolab",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha test"
+    "lint": "eslint test/*.js lib/*.js *.js",
+    "test:watch": "npm run test -- -w",
+    "test": "node index.js && ./node_modules/mocha/bin/mocha test"
```

Comment:
Instead of cluttering, what if we move `node index.js` to `"pretest"` instead of keeping it in `"test"`?